.github/2.0/cookbooks/1vs2.md

```diff
@@ -51,16 +101,65 @@ Benchmark #1: python -c "from jina import Document, Flow, Executor"
    with f:
        f.index(random_docs(10000))
 ```
+
+
+<table>
+<tr>
+<td>
+<b>2.0.0rc1</b>
+</td>
+</tr>
+<tr>
+<td>
+
 ```console
 ✅ done in ⏱ 8 seconds 🐎 1194.1/s
 ```
-- 1.2.4
+
+
+</td>
+</tr>
+<tr>
+<td>
+<b>1.4.2</b>
+</td>
+</tr>
+<tr>
+<td>
+
 ```console
 ✅ done in ⏱ 8 seconds 🐎 1127.9/s
 ```
 
+
+</td>
+</tr>
+</table>
+
 </details>
 
-1. Keyword 3
-1. Keyword 4
-1. Keyword 5
\ No newline at end of file
+## 2.0 Gives YOU the Power
+
+Jina 1.0 was a complex beast with lots of boilerplate code and not so much transparency. In 2.0 we're putting the power back in your hands. Want to write your own `.logger`, `.save()`, `.load()` or context manager interface for your Executor? Now that power is yours.
```

Comment:
the context manager interface I am not sure is accurate. I think we already expose __enter__ but u can redefine __exit__? Have to check code again

Reply:
I removed it for now